File: ios/stereoCamera/stereoCamera/Comm/Commands/SendZoom.swift
//
//  SendZoom.swift
//  stereoCamera
//
//  Created by hallmarklabs on 7/6/18.
//  Copyright © 2018 cody. All rights reserved.
//

import Foundation

class SendZoom : SetZoom
{
    override func doInit()
    {
        super.doInit()
        
        cmdtype = CommandTypes.RECEIVE_ZOOM
        expectsResponse = false
    }
}

File: android/app/src/main/java/com/munger/stereocamera/ip/command/commands/SendZoom.java
package com.munger.stereocamera.ip.command.commands;

import com.munger.stereocamera.ip.command.Command;

public class SendZoom extends SetZoom
{
	public SendZoom()
	{
		super();
	}

	public SendZoom(float value)
	{
		super(value);
	}

	@Override
	protected void doInit()
	{
		super.doInit();
		cmdtype = Type.RECEIVE_ZOOM;
		expectsResponse = false;
	}
}
